.gitmodules

```diff
@@ -16,8 +16,8 @@
 	branch = master
 [submodule "CMEPS"]
 	path = CMEPS
-	url = https://github.com/NOAA-EMC/CMEPS.git
-	branch = emc/develop
+	url = https://github.com/aerorahul/CMEPS.git
```

Comment:
Sorry if I missed something, is it intentional to point to @aerorahul's CMEPS repo in the merged version of the code?

Reply:
No. not intentional.

Reply:
There is a PR in CMEPS. https://github.com/NOAA-EMC/CMEPS/pull/16
Are they waiting for that one to merge?
I don't know how this works across repos.

Reply:
sorry I missed that merge. It is ready now.

Reply:
Usually it works the other way round, from bottom (innermost nested submodule) to top. Someone merges cmeps, you update your submodule pointer in your ufs-s2s-model PR to point back to the (now merged) official cmeps repo, then merge ufs-s2s-model.

Reply:
@climbfuji 
What do you want me to do now?
Should I open another PR in ufs-s2s-model with the updated .gitmodules and the submodule pointer?

Reply:
@aerorahul Please submit a new PR. Thanks!

Reply:
New PR issued. See https://github.com/ufs-community/ufs-s2s-model/pull/205